tests/epoch.rs

```diff
@@ -825,6 +825,9 @@ fn test_format() {
                     TimeScale::TDB => format!("{epoch:e}"),
                     TimeScale::TT => format!("{epoch:X}"),
                     TimeScale::UTC => format!("{epoch}"),
+                    TimeScale::GPST => format!("{epoch}"),
```

Comment:
I think this tests will fail because there is no shortcut to print in GPST, GST, or BDT, but the initialization from these epochs will set the time scale correctly. Let's see, maybe I'm wrong.